boards/ssrc/saluki/nuttx-config/nsh/defconfig

```diff
@@ -103,6 +107,9 @@ CONFIG_MAX_TASKS=64
 CONFIG_MCU_SERIAL=y
 CONFIG_MM_DEFAULT_MANAGER=y
 CONFIG_MM_REGIONS=1
+CONFIG_MMCSD=y
+CONFIG_MMCSD_SDIO=y
+CONFIG_MPFS_EMMCSD=y
```

Comment:
I need to explicitly set CONFIG_MMCSD_MMCSUPPORT=n
to get Kingston SDCS2/32GB microsd card to work.

Driver tries to identify card with MMC_CMD1 which is not supported and causes timeout.